components/event/CreateEvent/Steps/CreateEventStep2.tsx

```diff
@@ -68,16 +72,19 @@ function CreateEventStep2() {
       }
       setEvent(_event)
       try {
+        setShowLoadingBackdrop(true)
         if (!user || !user._id) {
           throw new Error("Invalid user. Please authenticate again.")
         }
         const savedEvent: Event | undefined = await saveEvent(_event, user._id)
         if (savedEvent) {
           resetEvent()
+          setShowLoadingBackdrop(false)
           router.push(`/event/detail/${savedEvent._id}`)
```

Comment:
Just to prevent error, can you use ?._id

Reply:
ok